### PR TITLE
Update the docs with the latest APIs

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change ###

Just updating the docs.

### Bugs Fixed ###

None.

### API Changes ###

None.

### Behavioral Changes ###

None.

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [ ] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### CODE REVIEW

File: docs/en/Xamarin.Essentials/SecureStorage.xml [hunk 1]
```diff
@@ -29,6 +29,24 @@
...
+        <AssemblyVersion>1.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>Security.SecAccessible</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>To be added.</summary>
+        <value>To be added.</value>
+        <remarks>To be added.</remarks>
+      </Docs>
```
Comment (mattleibow) [inline docs/en/Xamarin.Essentials/SecureStorage.xml]: This needs docs still.

File: docs/en/Xamarin.Essentials/SecureStorage.xml [hunk 2]
```diff
@@ -51,6 +69,30 @@
...
+        <Parameter Name="key" Type="System.String" />
+        <Parameter Name="accessible" Type="Security.SecAccessible" />
+      </Parameters>
+      <Docs>
+        <param name="key">To be added.</param>
+        <param name="accessible">To be added.</param>
+        <summary>To be added.</summary>
+        <returns>To be added.</returns>
+        <remarks>To be added.</remarks>
+      </Docs>
```
Comment (mattleibow) [inline docs/en/Xamarin.Essentials/SecureStorage.xml]: This needs docs still.

File: docs/en/Xamarin.Essentials/SecureStorage.xml [hunk 3]
```diff
@@ -75,5 +117,31 @@
...
+        <Parameter Name="value" Type="System.String" />
+        <Parameter Name="accessible" Type="Security.SecAccessible" />
+      </Parameters>
+      <Docs>
+        <param name="key">To be added.</param>
+        <param name="value">To be added.</param>
+        <param name="accessible">To be added.</param>
+        <summary>To be added.</summary>
+        <returns>To be added.</returns>
+        <remarks>To be added.</remarks>
```
Comment (mattleibow) [inline docs/en/Xamarin.Essentials/SecureStorage.xml]: This needs docs still.

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/en/FrameworksIndex/xamarin-essentials-android.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/FrameworksIndex/xamarin-essentials-android.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-ios.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/FrameworksIndex/xamarin-essentials-ios.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-uwp.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/FrameworksIndex/xamarin-essentials-uwp.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/FrameworksIndex/xamarin-essentials.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/Locale.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/Xamarin.Essentials/Locale.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/SecureStorage.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/Xamarin.Essentials/SecureStorage.xml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C8%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201805081648556109-233%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1zxsl7bLOPxHpuybJZcgNwFLbHZu0JtEazjHLhxXZ4s%3D&st=2018-05-08T16%3A45%3A00Z&se=2018-06-08T16%3A50%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/en/Xamarin.Essentials/SecureStorage.xml (hunk 1 above)
Comment (Redth) [reply docs/en/Xamarin.Essentials/SecureStorage.xml]: How about something like:

```
Provides the [SecAccessible](https://developer.xamarin.com/api/type/Security.SecAccessible/) level to use on iOS for subsequent calls to `GetAsync` and `SetAsync`
```

**Comment by dend** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/en/FrameworksIndex/xamarin-essentials-android.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/FrameworksIndex/xamarin-essentials-android.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-ios.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/FrameworksIndex/xamarin-essentials-ios.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials-uwp.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/FrameworksIndex/xamarin-essentials-uwp.xml) | :white_check_mark:Succeeded | |
[docs/en/FrameworksIndex/xamarin-essentials.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/FrameworksIndex/xamarin-essentials.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/Locale.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/Xamarin.Essentials/Locale.xml) | :white_check_mark:Succeeded | |
[docs/en/Xamarin.Essentials/SecureStorage.xml](https://github.com/xamarin/Essentials/blob/feature/update-docs/docs/en/Xamarin.Essentials/SecureStorage.xml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C11%5Cf63aadd0-c8f5-13ca-04e7-037e45f8c0b9%5CPullRequest%5C201805111639377791-233%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3ZacNwuEzG7dho0GnuZLGZfVu7OdL8BsKk44vYHPmEU%3D&st=2018-05-11T16%3A35%3A41Z&se=2018-06-11T16%3A40%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**